### DISCUSSION TITLE
How to keep Code-server running while browser closed

### DISCUSSION THREAD

**Discussion by weixinfeng88** [coder/code-server#6932] [General]
I installed code-server on my Mac and use my iPad for remote access. However, when I start a model training session and refresh the browser, the session stops. How can I keep it running? Thank you

**Comment by code-asher** [comment]
I believe this is a limitation of VS Code web, it kills the extension host and eventually terminals too when the browser closes.

If you can run it in a multiplexer like tmux or screen, that will make it stick around.  If it is an extension though there might not be anything you can do about it, it would need to be fixed in VS Code or the plugin itself.

**Reply by weixinfeng88** [reply]
Thx for the respond, yeah I try tmux, the problem once I reload my browser, the Jupyter notebook will stuck over there and can not run any block successfully

**Reply by code-asher** [reply]
Ohh I see, that sounds like potentially a bug with the Jupyter extension.  If you can reproduce it in Codespaces you could try reporting it in the extension's repo.
